tests/integration/views/test_static.py

```diff
@@ -0,0 +1,11 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestStatic(IntegrationTestCase):
+    def test_contact(self):
+        self.get('/contact-us')
+        self.assertInPage('Contact us')
```

Comment:
These asserts are not specific enough as these values are in the footer, even on the error pages